packages/dds/tree/src/simple-tree/api/viewSchemaToSimpleSchema.ts

```diff
@@ -43,7 +43,9 @@ export function toSimpleTreeSchema(schema: ImplicitFieldSchema): SimpleTreeSchem
 
 	// Include the "description" property only if it's present on the input.
 	if (normalizedSchema.metadata?.description !== undefined) {
-		output.description = normalizedSchema.metadata.description;
+		output.metadata = {
```

Comment:
For what it's worth, I think it is completely reasonable to copy the entire metadata bag from the view schema to the simple schema (rather than just copying the description over). We just can't copy over the entire bag when transforming from simple schema to json schema (since there may be JSON-incompatible / JSON-Schema-incompatible data there).

Reply:
These changes also seem like a great candidate for splitting out into a separate / more-scoped PR, since they don't depend on any of the other changes made here.